### ISSUE TITLE
MatcherOf.java:49-51: Remove all constructors except the...

### ISSUE THREAD

**Issue by 0pdd** [llorllale/cactoos-matchers#165]
The puzzle `135-e0b9c886` from #135 has to be resolved: 

https://github.com/llorllale/cactoos-matchers/blob/a40df7d674cf093ac3207b22beb2a2bc5913cd54/src/main/java/org/llorllale/cactoos/matchers/MatcherOf.java#L49-L51

The puzzle was created by @rultor on 17-Jul-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@bb368ec0-5f42-44b8-acc9-d78097fdae6e, thread: pool-1553-thread-1
org.hamcrest.MatcherAssert[20] java.lang.AssertionError: HTTP response status is not equal to 201:
404 Not Found [https://api.github.com/repos/llorllale/cactoos-matchers/labels]
Server: GitHub.com
Date: Fri, 17 Jul 2020 16:18:16 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4612
X-RateLimit-Reset: 1595004155
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
Vary: Accept-Encoding, Accept, X-Requested-With
X-GitHub-Request-Id: D594:760D:11E6C37:1FB7EA2:5F11CF48

{"message":"Not Found","documentation_url":"https://developer.github.com/v3/issues/labels/#create-a-label"}
Expected: HTTP response with status 201
     but: was <404 Not Found [https://api.github.com/repos/llorllale/cactoos-matchers/labels]
Server: GitHub.com
Date: Fri, 17 Jul 2020 16:18:16 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4612
X-RateLimit-Reset: 1595004155
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
Vary: Accept-Encoding, Accept, X-Requested-With
X-GitHub-Request-Id: D594:760D:11E6C37:1FB7EA2:5F11CF48

{"message":"Not Found","documentation_url":"https://developer.github.com/v3/issues/labels/#create-a-label"}>
```

[0.54.5](https://github.com/zerocracy/farm): Issue: llorllale/cactoos-matchers#165, Action: `opened`

**Comment by victornoel** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/llorllale/cactoos-matchers/issues/165#issuecomment-660224319))

@victornoel Job [#165](https://github.com/llorllale/cactoos-matchers/issues/165) is already in scope

**Comment by 0crat** [comment]
The job [#165](https://github.com/llorllale/cactoos-matchers/issues/165) assigned to @ruca905[/z](https://www.0crat.com/u/ruca905), here is [why](https://www.0crat.com/footprint/C63314D6Z/1afc3d02-8e89-49db-8808-b9e0ed1ac2b8); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by ruca905** [comment]
@0crat wait #138

**Comment by victornoel** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/llorllale/cactoos-matchers/issues/165#issuecomment-751455335))

@victornoel The user @ruca905[/z](https://www.0crat.com/u/ruca905) resigned from [#165](https://github.com/llorllale/cactoos-matchers/issues/165), please stop working. Reason for job resignation: Order was cancelled

**Comment by victornoel** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/llorllale/cactoos-matchers/issues/165#issuecomment-761098625))

@victornoel The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#165](https://github.com/llorllale/cactoos-matchers/issues/165), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
The job [#165](https://github.com/llorllale/cactoos-matchers/issues/165) assigned to @baudoliver7[/z](https://www.0crat.com/u/baudoliver7), here is [why](https://www.0crat.com/footprint/C63314D6Z/befd286d-c269-4635-bb7f-d9ff4a41ed6f); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0pdd** [comment]
The puzzle `135-e0b9c886` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#224](https://github.com/llorllale/cactoos-matchers/issues/224), [#225](https://github.com/llorllale/cactoos-matchers/issues/225) are still not solved.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @baudoliver7[/z](https://www.0crat.com/u/baudoliver7), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0pdd** [comment]
@0pdd the puzzle [#224](https://github.com/llorllale/cactoos-matchers/issues/224) is still not solved; solved: [#225](https://github.com/llorllale/cactoos-matchers/issues/225).
